docs/README.md

```diff
@@ -1,6 +1,6 @@
 ---
 {
-  "title": "MTGJSON",
+  "title": "Portable formats for Magic: The Gathering card data",
   "meta": [
     {
       "name": "description",
```

Comment:
Isn't that more of a description and should replace that, while the title stays MTGJSON?

Reply:
Yes and No. Yes it is, but no its not when thats the first thing people see as we are the top result for google.com

Reply:
You do bring up a good point about refining our SEO which I will open a ticket for.